components/scripts/tool-dropdown.tsx

```diff
@@ -14,6 +14,7 @@ import { GoTools } from 'react-icons/go';
 import { load } from '@/actions/gptscript';
 import { gatewayTool } from '@/actions/knowledge/util';
 import { ToolReference } from '@gptscript-ai/gptscript';
+import { ToolActionChatMessage } from '../shared/tools/ToolActionChatMessage';
```

Comment:
nit: I think most of our imports use `@/` rather than relative paths. If this can be changed to that, I think that would be good.